compiler-plugin/src/test/kotlin/arrow/meta/quotes/pbt/QuoteTest.kt

```diff
@@ -1,47 +1,73 @@
 package arrow.meta.quotes.pbt
 
+import arrow.meta.plugin.testing.Code
+import arrow.meta.plugin.testing.CompilerTest
+import arrow.meta.plugin.testing.assertThis
+import arrow.meta.quotes.scope.templates.BinaryExpressionTest.Companion.binaryExpression
+import arrow.meta.quotes.scope.templates.BlockExpressionTest.Companion.blockExpression
+import arrow.meta.quotes.scope.templates.BreakExpressionTest.Companion.breakExpression
+import arrow.meta.quotes.scope.templates.CatchClauseTest.Companion.catchClauseExpression
+import arrow.meta.quotes.scope.templates.ClassBodyTest.Companion.classBodyExpressions
+import arrow.meta.quotes.scope.templates.ContinueExpressionTest.Companion.continueExpressions
+import arrow.meta.quotes.scope.templates.DotQualifiedExpressionTest.Companion.dotQualifiedExpressions
+import arrow.meta.quotes.scope.templates.ForExpressionTest.Companion.forExpressions
+import arrow.meta.quotes.scope.templates.FunctionLiteralTest.Companion.functionalLiteralExpressions
+import arrow.meta.quotes.scope.templates.IfExpressionTest.Companion.ifExpressions
+import arrow.meta.quotes.scope.templates.ImportDirectiveTest.Companion.importDirectiveExpression
+import arrow.meta.quotes.scope.templates.IsExpressionTest.Companion.isExpressions
+import arrow.meta.quotes.scope.templates.ObjectDeclarationTest.Companion.objectDeclarationExpression
+import arrow.meta.quotes.scope.templates.PackageDirectiveTest.Companion.packageExpressions
+import arrow.meta.quotes.scope.templates.ReturnExpressionTest.Companion.returnExpressions
+import arrow.meta.quotes.scope.templates.ThrowExpressionTest.Companion.throwExpression
+import arrow.meta.quotes.scope.templates.TryExpressionTest.Companion.tryExpression
+import arrow.meta.quotes.scope.templates.TypeAliasTest.Companion.typeAliasExpressions
+import arrow.meta.quotes.scope.templates.TypeReferenceTest.Companion.typeReferenceExpression
+import arrow.meta.quotes.scope.templates.WhenConditionTest.Companion.whenConditionExpression
+import arrow.meta.quotes.scope.templates.WhenEntryTest.Companion.whenEntryExpression
+import arrow.meta.quotes.scope.templates.WhenExpressionTest.Companion.whenExpressions
+import arrow.meta.quotes.scope.templates.WhileExpressionTest.Companion.whileExpression
 import org.junit.Test
 
 class QuoteTest {
 
   @Test
   fun `Check identity law`() {
 
-//    listOf(
-//      binaryExpression,
-//      blockExpression,
-//      breakExpression,
-//      *forExpressions,
-//      catchClauseExpression,
-//      *classBodyExpressions,
-//      *continueExpressions,
-//      // destructuringDeclarationExpression, // TODO: data class
-//      *dotQualifiedExpressions,
```

Comment:
Hi  @matiaslev , thanks for this awesome contribution!! :tada: Please, don't uncomment this file because it's still a WIP and it could lead to confusion. Thanks!!

Reply:
Please let me know if I can help in some way,
What I am trying to do is organize the unit tests in a way that the QuoteTest can consume the same Code.Source 's.
did you think that what I did make sense but will be better merge it without uncommenting this file?

Reply:
it's commented again :)

Reply:
Thanks @matiaslev !! :tada: This contribution makes sense, of course, thanks!! 

That file is a WIP to test the identity law. However, maybe we could do other improvements with these tests. I'll let you know if we come up with more ideas :+1: